ownCloud/Client/PhotoSelectionViewController.swift

```diff
@@ -0,0 +1,337 @@
+//
+//  PhotoUploadViewController.swift
+//  ownCloud
+//
+//  Created by Michael Neuwert on 24.02.2019.
+//  Copyright © 2019 ownCloud GmbH. All rights reserved.
+//
+
+import UIKit
+import Photos
+import PhotosUI
+
+private extension UICollectionView {
+	func indexPathsForElements(in rect: CGRect) -> [IndexPath] {
+		let allLayoutAttributes = collectionViewLayout.layoutAttributesForElements(in: rect)!
+		return allLayoutAttributes.map { $0.indexPath }
+	}
+}
+
+typealias PhotosSelectedCallback = ([PHAsset]) -> Void
+
+class PhotoSelectionViewController: UICollectionViewController, Themeable {
+
```

Comment:
would be nice, to mark this as `// MARK: - Constants` section

Reply:
@hosy done